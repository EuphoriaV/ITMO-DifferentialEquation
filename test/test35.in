1.97
((3.74 * (x * x)) + (((x + 10.4) - (((x + 4.83) * (x + x)) * 9.92)) + (((x - x) * (8.19 * x)) * x)))
2.28
2.31