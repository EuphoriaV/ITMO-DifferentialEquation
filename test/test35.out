-2.1532465